src/stream_fold.rs

```diff
@@ -12,41 +12,41 @@
 // See the License for the specific language governing permissions and
```

Comment:
This module can probably be replaced by `futures`' implementation in a subsequent PR.

Reply:
Are you referring to `futures::stream::fold` ? I actually just noticed it existed, converting it without a significant rewrite would be tricky due to how `StreamFold` currently returns the stream which is not possible with `Pin`. Good info!

Reply:
Okay. I didn't look at this module in detail. If it's hard to replace, let's keep it around for now.